README.md

```diff
@@ -1,47 +1,127 @@
-# Project-Pride-Land
+# 🌻 Project-Pride-Land
 
-## Introduction
+## 🐐 Goal
+This is the front-end repository for our project Pride Farm! Our mission is to assist the farm NPO ぷらいどらんど in making a website for its community;
+a place where anyone and everyone can come together and particpate in farming regardless of their health conditions.
 
-## Built with:
+## 💬 Mission
+We want to be able to express what ぷらいどらんど stands for and ensure that their message of companionship, acceptance, and community is displayed. Not only that, but we also strive to make this website easy to maintain and understand for the staff at the NPO.
+
+## 💻 Built with:
 
 Our Front-End Tech stack: <br>
 
 ```
 • React-Vite
 • Typescript
 • TailwindCSS
+• PHASER
 ``` 
+# 👷 Contributing
+Make sure to fork our repo, and fill out the informaiton in the PR message template! If you have any questions feel free to reach out to any of our members.
```

Comment:
Why are they forking your repo and not cloning it? Also information has a typo 😊